io.github.rsevero.mapiah.yml

```diff
@@ -0,0 +1,54 @@
+# Generated from flatpak-flutter.yml, do not edit
+# Visit the flatpak-flutter project at https://github.com/TheAppgineer/flatpak-flutter
+id: io.github.rsevero.mapiah
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm19
+command: mapiah
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+cleanup:
+  - /var/lib
```

Comment:
I don't think this has any effect.

Reply:
> The cleanup is not necessary? 

If you actually read the documentation it will tell you that cleanup rules are always inside `$FLATPAK_DEST` which mean that it will never touch `/var/lib`.